src/commands/createStaticWebApp/GitHubOrgListStep.ts

```diff
@@ -27,17 +27,29 @@ export class GitHubOrgListStep extends AzureWizardPromptStep<IStaticWebAppWizard
     }
 
     public shouldPrompt(context: IStaticWebAppWizardContext): boolean {
-        return !context.repoHtmlUrl || context.createScenario === 'publishToNewRepo';
+        // if orgData was set earlier, we should push the value to mask
+        if (context.orgData) {
+            context.valuesToMask.push(context.orgData.login);
```

Comment:
Yep, both are needed.  On L26, that's happening if the user selected something during the `prompt` part of the step.  But L32 is happening in the `shouldPrompt` part of the step, which executes prior to `prompt` to see if `prompt` should execute or not.

The reason for this change is because we default the user to their AuthenticatedUser in basic create rather than letting them select an org, so if we auto-default, we should still push the username into `valuesToMask`.

I could have done this at the time of setting orgData, but I thought it was more appropriate here.

Reply:
I see - I was concerned that it would double-push to `valuesToMask` but I see now that that won't happen